packages/backend/src/services/specimenService/models/relations.js

```diff
@@ -24,8 +24,8 @@ const relationships = [
   {
     sourceResource: 'specimen',
     type: 'belongsToMany',
-    targetAs: 'taxa',
-    targetResource: 'taxon',
+    targetAs: 'taxonNames',
```

Comment:
this is only needed if the plural is different than `${singular}s` :-) but this works too ofc